src/test/java/world/bentobox/bentobox/listeners/flags/protection/PhysicalInteractionListenerTest.java

```diff
@@ -193,6 +195,10 @@ public void setUp() {
 
         // Util strip spaces
         when(Util.stripSpaceAfterColorCodes(anyString())).thenCallRealMethod();
+
+        // Tags
+        when(Tag.PRESSURE_PLATES.isTagged(any(Material.class))).thenReturn(true);
+        when(Tag.WOODEN_BUTTONS.isTagged(any(Material.class))).thenReturn(true);
     }
```

Comment:
I've always had such problems trying to mock Tags. If this works, I'm kicking myself!

Reply:
It works now. I might have broken my desk if it didn't 🤣.